shell/utils/formatter.js

```diff
@@ -0,0 +1,12 @@
+
+const OPAQUE_SECRET_TYPE = 'Opaque';
+
+export function formatEncryptionSecretNames(secrets, chartNamespace) {
```

Comment:
If you are entirely aware of the whole logic, could you add a comment to the function? If it's just a abstraction and your knowledge is _opaque_ (😆), which happens to everyone and very often in our code 😓, we may avoid to add the comment for now.

Reply:
We were trying to figure out where this requirement came from, but we couldn't trace it. So, we basically just know that at some point a couple years ago it was decided that we should only show opaque secrets. I doubt a comment will be helpful, unfortunately